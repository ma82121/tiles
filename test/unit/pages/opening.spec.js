import { shallowMount, createLocalVue } from '@vue/test-utils'
import VueRouter from 'vue-router'
import Opening from '@/pages/opening'

const localVue = createLocalVue()
localVue.use(VueRouter)
const router = new VueRouter()

describe('Opening', () => {
  test('クリックした時にdata.clickedがtrueになる', () => {
    const wrapper = shallowMount(Opening, {
      localVue,
      router
    })
    wrapper.find('div').trigger('click')
    // expect(wrapper.vm.clicked).toBe(true)
  })
  test('snapshot', () => {
    const wrapper = shallowMount(Opening, {
      localVue,
      router
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
