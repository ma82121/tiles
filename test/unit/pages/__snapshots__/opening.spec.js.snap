// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Opening renders correctly 1`] = `
<div
  class="opening"
>
  <tileslogo-stub
    hasanimation="true"
  />
</div>
`;

exports[`Opening snapshot 1`] = `
<div
  class="opening"
>
  <tileslogo-stub
    hasanimation="true"
  />
</div>
`;
